docs/Product-Guides/Assisted-Connect-Flow.md

```diff
@@ -1,45 +1,42 @@
 # Assisted Connect Flow
 
-Assisted Connect Flow (ACF) is our semi-automated product that expands coverage to 70+ providers. In contrast to our automated connections, the data for ACF providers is refreshed periodically. Our full list of coverage — for both API and ACF providers — can be found [here](../Development-Guides/Providers.md).
+Assisted API Connections is our semi-automated product that expands coverage to 70+ providers. In contrast to our [Automated API Connections](../Product-Guides/Automated-Connect-Flow.md), the data for Assisted API connections is refreshed periodically (usually every 14 days). Our full list of coverage — for both Automated API and Assisted API providers — can be found [here](../Development-Guides/Providers.md).
```

Comment:
1- I think this flow is still referred to as "ACF" or "Assisted Connect Flow". If we want to change that, it might require some more communication to Ops and Customers because I _think_ that's actually a term that external folks might be familiar with using now.  To rework the start of this paragraph, maybe something like...

```
Assisted Connect Flow (ACF) is our semi-automated product that expands coverage to 70+ providers using Assisted API Connections .
```

What do you think?


2- Unless we document it in other places as well, I think I'd refrain from adding the bit about 14 days.

Reply:
I pulled the "14 days"  from some text somewhere, but I can't remember, so I can remove it for now.

Reply:
Assisted Connect Flow just doesn't work if we are trying to rebrand our Automated API connections sections as well. Connect and Assisted Connect (the way they are both worded today on our docs) just doesn't make a lot of sense. Connect (what we are calling Automated API Connections) really clashes with our naming of Finch Connect. We should clarify on names. 

Finch Connect (single product) either uses Automated API Connections or Assisted API Connections. Which is why I was proposing renaming them to Automated Connect Flow and Assisted Connect Flow in our docs.

Reply:
I resolved #2.

Reply:
Got it, okay I'm with you.